### DISCUSSION TITLE
2023/12/08/%E6%9B%B4%E6%96%B0-5/

### DISCUSSION THREAD

**Discussion by giscus** [FloranceYeh/FloranceYeh.github.io#17] [Announcements]
# 2023/12/08/%E6%9B%B4%E6%96%B0-5/

It's Florance!

https://florance.eu.org/2023/12/08/%E6%9B%B4%E6%96%B0-5/

**Comment by amydxhs** [comment]
所以直到现在你都还没有开始做

**Comment by snoWxc** [comment]
主页背景很漂亮，是博主生成的吗，还是说是网图

**Reply by FloranceYeh** [reply]
是P站上 **つね** 这位太太画的

**Reply by snoWxc** [reply]
好的，多谢
